### ISSUE TITLE
After update to 2.3.1 wallabag can't retrieve contents for any article

### ISSUE THREAD

**Issue by taosxx** [wallabag/wallabag#3552]
### Issue details

After the update to 2.3.1, I can no longer fetch new articles or re-fetch those that have been added with an older version.
If I add a link via the "+" icon on the top bar (or with the bookmarklet), the article is saved under the title "No title found" and the content is empty except for "wallabag can't retrieve contents for this article. Please troubleshoot this issue. "
With wallabagger I get a title that corresponds to the last part of the article url and the content is completely empty.
Tagging, annotating and exporting work without a problem.

I've enabled debug logging by editing app/config/config_prod.yml but can't find out what the underlying problem is:
[prod.log](https://github.com/wallabag/wallabag/files/1609240/prod.log)

In addition, I did a new installation (incl. a new empty database) via `git clone` - `make install` for 2.3.1 on the same server but didn't see any difference.

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.3.1
* How did you install wallabag? Via `git clone` or by downloading the package? `git clone` (initially 2.2.3, then update via `make update` to 2.3.0 and then 2.3.1)
* Last wallabag version that did not exhibit the issue (if applicable): 2.3.0
* php version: 7.1.12
* OS: Fedora 27 (updated from 26)
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): SQLite (default for 2.2.3)

### Steps to reproduce/test case

?

**Comment by j0k3r** [comment]
Looks like the website you are trying to fetch is generating a HTTP error and wallabag can't fetch it:

> [2018-01-06 02:44:10] graby.WARNING: Request throw exception (with no response): cURL Error:  {"error_message":"cURL Error: "} []

But that article works find on my side

Could you try that on your server:

```
curl -SIv https://arstechnica.com/gadgets/2018/01/meltdown-and-spectre-heres-what-intel-apple-microsoft-others-are-doing-about-it/
```

**Comment by taosxx** [comment]
Here's what I see:

[answer.txt](https://github.com/wallabag/wallabag/files/1609692/answer.txt)

**Comment by j0k3r** [comment]
I don't know then.
Every article you tried to add fail?

**Comment by taosxx** [comment]
Every single one.

The last one I could successfully retrieve was a test after the update to 2.3.0 to see if everything still worked.

**Comment by j0k3r** [comment]
I guess it's sth on your server installation / wallabag configuration rather than a wallabag bug.
But I don't know what to ask more to find the problem. :confused:

**Comment by taosxx** [comment]
Okay, so I've done another fresh installation of 2.3.1, this time with MariaDB instead of SQLite.  I still can't fetch articles so I assume you're right that there is something strange going on with my server. Any ideas - maybe some strange permissions in /var/... I could look for?

**Comment by taosxx** [comment]
Just updated to Fedora 28 but still see the same issue.

**Comment by taosxx** [comment]
Okay, that's strange:

I've installed a new virtual test server based on Fedora 28 and decided to see if I could get wallabag to run. So, I installed wallabag 2.3.2 from the git repo, ran `make install` as apache user, and started a test run (also as apache user) with `make run`. I used lynx to login and added a few articles - everything worked fine and I could fetch the content of each page. Nice!

Then I went back to my real server and started a test run (`make run` as apache user). Again, I tried to add an article with lynx but couldn't get it to fetch the content of the page. Okay, that's the behaviour I've already known since the update to version 2.3.1.
I started another test run, this time as root. Logged in via lynx and re-fetched successfully the content of every single article. That was unexpected (at least for me).

Any idea what's going on my "real" server? Is there a permission I have to change somewhere (I've gone over and over the instructions in the installation manual, [Rights access to the folders of the project](https://doc.wallabag.org/en/admin/installation/rightaccess.html), but couldn't see anything I haven't already done and checked several times.
